Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -897,17 +897,17 @@
   - type: ExplodeOnTrigger
   - type: Explosive
     explosionType: Default
-    totalIntensity: 30 # Delta-V
-    intensitySlope: 1
+    totalIntensity: 150 # DeltaV
+    intensitySlope: 2 # DeltaV
     maxIntensity: 5
     canCreateVacuum: false
   - type: ContainerContainer
     containers:
       cluster-payload: !type:Container
   - type: ProjectileGrenade
-    fillPrototype: PelletClusterLethal # Delta-V
-    capacity: 15 # Delta-V
-    triggerKey: trigger # Delta-V
+    fillPrototype: PelletClusterLessLethal
+    capacity: 30
+    triggerKey: trigger # DeltaV
```

This is fixed in upstream so just change this portion to reflect this.

```suggestion
    triggerKey: trigger
    fillPrototype: PelletClusterLessLethal
    capacity: 30
```

---

gotcha gotcha